plugins/node/opentelemetry-instrumentation-aws-lambda/test/integrations/lambda-handler.test.ts

```diff
@@ -37,6 +37,7 @@ import {
   SEMATTRS_EXCEPTION_MESSAGE,
   SEMATTRS_FAAS_COLDSTART,
   SEMATTRS_FAAS_EXECUTION,
+  SEMATTRS_HTTP_URL,
```

Comment:
It is better to use `ATTR_URL_FULL` new constant instead. cc @dyladan

Reply:
Happy to do either way (old or new) - I noted above that I didn't see any thing else using the new semantic convention in this repo (or otel-js for that matter).

Reply:
Yeah, I know. Current code base is using the constant family you used. 

But I was asked by @dyladan to use the new constant in my previous PR (https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2403), so I think the approach is using the new constants for new attributes additions here and migrate the existing one some time later.

Reply:
OK, cool, I'll make the switch.